<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConversionDialog</class>
 <widget class="QDialog" name="ConversionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cartesian &lt;&gt; Proj</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0,0,0">
   <item row="2" column="2" colspan="2">
    <widget class="QLineEdit" name="CORFile"/>
   </item>
   <item row="4" column="4">
    <widget class="QPushButton" name="pushButtonXYZ">
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLabel" name="labelNbPts">
     <property name="text">
      <string>Points processed: 0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Projected files (COR, NEW)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="pushButtonCOR">
     <property name="minimumSize">
      <size>
       <width>10</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="QLineEdit" name="XYZFile">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Cartesian files (TXT, XYZ)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Number of significant digits</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QSpinBox" name="digitsSpinBox">
     <property name="maximum">
      <number>20</number>
     </property>
     <property name="value">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QPushButton" name="toCartesianButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>&gt;&gt;To Cartesian</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QPushButton" name="toProjButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>&gt;&gt;To Proj</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="5">
    <widget class="FrameWidget" name="groupBoxFrame">
     <property name="title">
      <string>Frame</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FrameWidget</class>
   <extends>QGroupBox</extends>
   <header>gui/framewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>CORFile</tabstop>
  <tabstop>XYZFile</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
